synapse/cores/common.py

```diff
@@ -147,7 +273,7 @@ def __init__(self, link, store, **conf):
 
         # perm defs are also used to define trigger metadata
         # they must be loaded prior to triggers being loaded
-        self._initPermDefs()
+        #self._initPermDefs()
```

Comment:
Should this line just be removed?

Reply:
fixed - this actually had to be updated to work properly on a auth core and ``test_cortex_aware`` was updated to account for triggers.`